Add README and Code-Signing Working Group Charter (#20)
cabforum/forum: README.md

## Overview

> This repository contains information related to the
> [CA/Browser Forum's](https://www.cabforum.org)
> [Plenary Organization](https://cabforum.org/about-us/) which governs the operations of the CABF as a whole.

Reviewer: I'm not aware of us ever using the term "Plenary Organization" - do you have some references?

I'm wondering if this might be better as:

```suggestion
This repository contains information related to the
[CA/Browser Forum](https://www.cabforum.org) and the charters of specific
Chartered Working Groups which have been approved by the Forum.
```

Author: I don't--it's something we've used on calls and things to differentiate the CABF umbrella call from the SCWG call, but it doesn't have any basis in the Bylaws. I agree it's better to stick with the Bylaws terms for now until we can review those (we might introduce that term at some point as we do some revisions to finish cleaning up from the WG split-up).

> In particular, the following documents are maintained by the CABF Plenary:

Reviewer: ```suggestion
In particular, this repository maintains documents that are voted on by the CA/Browser Forum as a whole:
```

* [The Bylaws of the CA/Browser Forum](https://cabforum.org/bylaws/)

>   Pandoc-flavored Markdown: [docs/BR.md](Bylaws.md)

Reviewer: ```suggestion
  Pandoc-flavored Markdown: [docs/Bylaws.md](Bylaws.md)
```

> * The charters of the various CA/Browser Forum Working Groups:

Reviewer: ```suggestion
* The charters of the various CA/Browser Forum Chartered Working Groups:
```

>   * The [Server Certificate Working Group](https://cabforum.org/working-groups/scwg/) Pandoc-flavored Markdown: [SCWG-charter.md](SCWG-charter.md)
>   * The [S/MIME Certificate Working Group](https://cabforum.org/working-groups/smime-certificate-wg/) Pandoc-flavored Markdown: [SMCWG-charter.md](SMCWG-charter.md)
>   * The [Code-Signing Working Group](https://cabforum.org/code-signing-working-group/) Pandoc-flavored Markdown: [CSCWG-charter.md](CSCWG-charter.md)

Reviewer: ```suggestion
  * The [Server Certificate Chartered Working Group](https://cabforum.org/working-groups/scwg/) Pandoc-flavored Markdown: [SCWG-charter.md](SCWG-charter.md)
  * The [S/MIME Certificate Chartered Working Group](https://cabforum.org/working-groups/smime-certificate-wg/) Pandoc-flavored Markdown: [SMCWG-charter.md](SMCWG-charter.md)
  * The [Code-Signing Chartered Working Group](https://cabforum.org/code-signing-working-group/) Pandoc-flavored Markdown: [CSCWG-charter.md](CSCWG-charter.md)
```
